enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/TemporalIndexAcceptanceTest.scala

```diff
@@ -42,6 +44,30 @@ class TemporalIndexAcceptanceTest extends IndexingTestSupport {
       DateValue.epochDate(10002),
       DateValue.epochDate(10004),
       DateValue.epochDate(10006))
+
+    assertRangeScan(
+      LocalDateTimeValue.localDateTime(10000, 100000000),
+      LocalDateTimeValue.localDateTime(10002, 100000000),
+      LocalDateTimeValue.localDateTime(10004, 100000000),
+      LocalDateTimeValue.localDateTime(10006, 100000000))
+
+    assertRangeScan(
+      DateTimeValue.datetime(10000, 100000000, ZoneId.of("Europe/Stockholm")),
+      DateTimeValue.datetime(10002, 100000000, ZoneId.of("Europe/Stockholm")),
+      DateTimeValue.datetime(10004, 100000000, ZoneId.of("Europe/Stockholm")),
+      DateTimeValue.datetime(10006, 100000000, ZoneId.of("Europe/Stockholm")))
+
+    assertRangeScan(
+      LocalTimeValue.localTime(10000),
+      LocalTimeValue.localTime(10002),
+      LocalTimeValue.localTime(10004),
+      LocalTimeValue.localTime(10006))
+
+    assertRangeScan(
+      TimeValue.time(10000, ZoneId.of("Europe/Stockholm")),
```

Comment:
Will the `ZoneId.of` return some zone offset? Because `TimeValue` should not work with named timezones...

Reply:
Weeeell, it used to convert into an offset later. We changed it to only permit offsets in the factory method as expected.